api/v1beta2/appwrapper_types.go

```diff
@@ -63,14 +63,21 @@ type AppWrapperStatus struct {
 type AppWrapperPhase string
 
 const (
-	AppWrapperEmpty      AppWrapperPhase = ""
-	AppWrapperSuspended  AppWrapperPhase = "Suspended"
-	AppWrapperDeploying  AppWrapperPhase = "Deploying"
-	AppWrapperRunning    AppWrapperPhase = "Running"
-	AppWrapperSuspending AppWrapperPhase = "Suspending"
-	AppWrapperDeleting   AppWrapperPhase = "Deleting"
-	AppWrapperCompleted  AppWrapperPhase = "Completed"
-	AppWrapperFailed     AppWrapperPhase = "Failed"
+	AppWrapperEmpty       AppWrapperPhase = ""
+	AppWrapperSuspended   AppWrapperPhase = "Suspended"
+	AppWrapperResuming    AppWrapperPhase = "Resuming"
+	AppWrapperRunning     AppWrapperPhase = "Running"
+	AppWrapperSuspending  AppWrapperPhase = "Suspending"
+	AppWrapperSucceeded   AppWrapperPhase = "Succeeded"
+	AppWrapperFailed      AppWrapperPhase = "Failed"
+	AppWrapperTerminating AppWrapperPhase = "Terminating"
+)
+
+type AppWrapperCondition string
+
+const (
+	ActiveResources  AppWrapperCondition = "ActiveResources"
+	PassiveResources AppWrapperCondition = "PassiveResources"
```

Comment:
As named these two conditions sound like opposites, which is not the intent. I suggest we rename the two conditions to make it obvious that:
- one condition is about quota reservation: if false then the appwrapper is not using quota,
- one condition is about deployed resources: if false then there are no wrapped objects deployed on the cluster.